src/PolymathBase.ts

```diff
@@ -561,4 +654,45 @@ export class PolymathBase extends PolymathAPI {
 
     return dividends;
   };
+
+  public roleToPermission = async ({ role }: { role: SecurityTokenRole }) => {
+    let moduleName: ModuleName;
+    let permission: Perm;
+
+    if (role === SecurityTokenRole.ShareholdersAdministrator) {
+      moduleName = ModuleName.GeneralTransferManager;
+      permission = Perm.Admin;
+    } else if (role === SecurityTokenRole.PermissionsAdministrator) {
+      moduleName = ModuleName.GeneralPermissionManager;
+      permission = Perm.Admin;
+    } else if (
+      [
+        SecurityTokenRole.Erc20DividendsAdministrator,
+        SecurityTokenRole.Erc20DividendsOperator,
+      ].includes(role)
+    ) {
+      moduleName = ModuleName.GeneralTransferManager;
+      permission =
+        role === SecurityTokenRole.Erc20DividendsAdministrator ? Perm.Admin : Perm.Operator;
+    } else if (
+      [
+        SecurityTokenRole.EtherDividendsAdministrator,
+        SecurityTokenRole.EtherDividendsOperator,
+      ].includes(role)
+    ) {
+      moduleName = ModuleName.ERC20DividendCheckpoint;
```

Comment:
Shouldn't this one be `ModuleName.EtherDividendCheckpoint`?

Reply:
Yeah, my bad